lib/top_hangman/game.rb

```diff
@@ -1,51 +1,30 @@
 require_relative 'round'
+require_relative 'word'
+require_relative 'render'
 
 module TopHangman
   class Game
-    MIN_WORD_SIZE = 5
-    MAX_WORD_SIZE = 12
-
-    attr_reader :base_words
-
-    def initialize(file_name)
-      @base_words = read_words_file(file_name)
-      @list_of_words_size = base_words&.size
-      @history_of_words = []
-      @round = create_round
+    def initialize
+      @word = Word.new
+      @render = Render.new
     end
 
-    private
-
-    def read_words_file(file_name)
-      return nil unless File.exist?(file_name)
-
-      base_file = File.open(file_name, 'r')
-      base_file.readlines.map(&:chomp).select { |line| line.size > MIN_WORD_SIZE && line.size < MAX_WORD_SIZE }
+    def start
+      game_loop
     end
 
-    def random_word
-      return nil unless base_words
-
-      word = base_words[rand(0..@list_of_words_size)]
-      return word unless word_in_history?(word)
-
-      random_word
-    end
+    def game_loop
+      @round = create_round
 
-    def word_in_history?(word)
-      @history_of_words.include?(word)
+      result_of_round = @round.start_round
+      @render.endgame_message(result_of_round)
```

Comment:
```suggestion
      round_result = @round.start_round
      @render.endgame_message(round_result)
```

Reply:
I don't think `@round.start_round` should return the round result.  `result` should be a separate attribute of the `round`